terraform/terraform_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'docker'
+require 'serverspec'
+
+describe "Terraform image" do
+    before(:all) {
+        set :docker_image, find_image_id('terraform:latest')
+    }
+
+    it "installs the Alpine linux" do
+        expect(file("/etc/alpine-release")).to be_file
+    end
+
+    it "installs Root Certificates" do
+        expect(file("/usr/share/ca-certificates/mozilla/GlobalSign_Root_CA.crt")).to be_file
+    end
+
+    it "checks if terraform binary is executable" do
+        expect(file("/usr/local/bin/terraform")).to be_mode 755
+    end
+
+    it "installs all the whitelisted binaries" do
+        command("echo $BINARY_WHITELIST").stdout.split.each { |f|
+            expect(file("/usr/local/bin/#{f}")).to be_mode 755
+        }
```

Comment:
>  It seems unzip will return a non-zero exit code if you specify a path that's not present in the archive. Does that not fail the docker build?

I have just checked, and it fails on my OS (ubuntu, UnZip 6.00 of 20 April 2009, by Debian. Original by Info-ZIP.) but not on the alpine one (BusyBox v1.24.1 (2015-12-16 08:00:02 GMT) multi-call binary.). 

So the test is kinda required.